realsense_camera/src/base_nodelet.cpp

```diff
@@ -723,8 +723,7 @@ namespace realsense_camera
       ROS_INFO_STREAM(nodelet_name_ << " - Starting camera");
       // Set up the callbacks for each stream
       setFrameCallbacks();
-      rs_start_device(rs_device_, &rs_error_);
-      checkError();
+      rs_device_->start(rs_source_);
```

Comment:
This is the only C++ style realsense call we are making. Was there no C api for this?
If this throws an exception (during start/stop power service), do we catch it?